### ISSUE TITLE
Fix ViewFeedback feature

### ISSUE THREAD

**Issue by cannotparse** [inspectorG4dget/AnonymousFeedback#29]
The view feedback feature is broken with the new schema changes. please fix

**Comment by cannotparse** [comment]
Here is the JSON I am sending 
```javascript
 data = {
				courseCode : $(".course_selecter").val() ,
				sectionCode : $(".section_selecter").val()
			};
```

**Comment by scriptbae** [comment]
From #35 (dupe, closed):

>Using the Professor view, the application fails to display collected feedback, or acknowledge a lack thereof. This feature should maybe be implemented as an HTML table.

>The root cause of this appears to be a bad unpacking of data fetched from the database by `getCourseFeedbacks()` in `dbhandler.py.`

**Comment by scriptbae** [comment]
Fixed in #47.
